pkg/translate/translate.go

```diff
@@ -58,6 +59,9 @@ func (t *TranslateServiceServer) UploadTranslationFile(
 	ctx context.Context,
 	req *pb.UploadTranslationFileRequest,
 ) (*emptypb.Empty, error) {
+	span := trace.SpanFromContext(ctx)
```

Comment:
Do spans have to be added manually? There weren't any OTEL wrappers for this purpose?

Reply:
No, it's not needed - the GRPC server should have one. https://github.com/translate-agent/translate/pull/24/files#diff-06a04b9bbf6fc3785a7a7389f59f4cf08c4133c69c665dd689072445a43d632aR44

Reply:
You are right